bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -73,6 +89,9 @@ spec:
           }
           trap deletePid EXIT
 
+          chown -R openvswitch:openvswitch /var/run/openvswitch
```

Comment:
About the line:
              kill -9 $(cat /var/run/openvswitch/ovs-vswitchd.pid 2>/dev/null) 2>/dev/null || true

ovsdb-server has by default defined on the systemd unit Restart=on-failure, a kill -9 would restart it. I haven't actually tried it but I think systemctl kill would get the job done.

Reply:
this is for containerized mode only - and is unchanged by this pr